recipes-graphics/spark/spark_git.bb

```diff
@@ -0,0 +1,100 @@
+include pxcore.inc
+
+DEPENDS_append = " curl freetype util-linux libjpeg-turbo libpng pxcore-libnode giflib sqlite3 wpeframework-tools-native wpeframework wpeframework-clientlibraries"
+
+SRC_URI += "file://Spark.pc \
```

Comment:
can you please use append here and move the first entry to a separete line?

Reply:
done